.github/CODEOWNERS

```diff
@@ -1 +1 @@
- *       @einride/team-awareness @einride/team-a2b
+ *       @einride/team-awareness @einride/team-a2b @odsod
```

Comment:
Perhaps we should remove the other 2 teams since they are a bit deprecated :thinking: ?

Reply:
Good point! How about handing it over to @einride/team-2lchain for now?